src/meta-client/src/client/router.rs

```diff
@@ -122,8 +123,15 @@ impl Inner {
     async fn delete(&self, mut req: DeleteRequest) -> Result<RouteResponse> {
         let mut client = self.random_client()?;
         req.set_header(self.id);
-        let res = client.delete(req).await.context(error::TonicStatusSnafu)?;
-
+        let res = client.delete(req).await.map_err(|mut source| {
+            // FIXME(hl): here intentionally clear the metadata field so that error date does not changes which will break sqlness test.
```

Comment:
It's [merged](https://github.com/CeresDB/sqlness/commits/main), I'll submit a follow-up patch to fix this